internal/app/app.go

```diff
@@ -79,11 +79,18 @@ type FileChange struct {
 	Patch       string `json:"patch"`
 }
 
+// PRReviewMap is a map of file paths to a list of PRCommentInfo
 type PRReviewMap map[string][]PRCommentInfo
 type PRCommentInfo struct {
 	CommentBody string
-	StartLine   int
-	Line        int
+	// The start_line is the first line in the pull request diff that your multi-line comment applies to.
+	StartLine int
```

Comment:
Consider adding more detailed comments to newly added constants, explaining their purpose or usage across the application.